### PR TITLE
Updated to Bootstrap v3 and fixed some bugs

### PR DESCRIPTION
**Pull request by olvlvl**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+.DS_Store
+.idea/
```
Comment (TomasVotruba) [inline .gitignore:2]: I'll remove this on merge, just FYI: these IDE files should be [ignored globally](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore), so every repo doesn't have to contain all IDE temp files.

**Comment by TomasVotruba** [comment]
Amazing job, thanks a lot! You saved me some precious time.

Is there any feature you'd like to see in ApiGen?

**Comment by olvlvl** [comment]
Well, currently I'm waiting for this PR to be merged https://github.com/Andrewsville/PHP-Token-Reflection/pull/60 because I can't generate doc for most of my projects. I would like to thank you Tomas for resuscitating ApiGen, I like it a lot.

File: .gitignore (hunk 1 above)
Comment (olvlvl) [inline .gitignore:2]: Thanks for the global-ignore, I didn't know about that.

**Comment by TomasVotruba** [comment]
My pleasure, seems like a good project with loads of support from community.

As for Token Reflection, I'm [working on a fork ATM](https://github.com/ApiGen/ApiGen/issues/495#issuecomment-77709254). Most work is done, now I have to connect and replace it in ApiGen.

**Comment by olvlvl** [comment]
Good to know.

**Pull request merged**